### ISSUE TITLE
[Feature Request] List or drop-down list of (sorted) subscribed communities.

### ISSUE THREAD

**Issue by BitingChaos** [rystaf/mlmym#39]
On reddit, my top bar has a drop-down list of my subscribed subreddits, Home (in bold), Popular, All, and Random. I can easily & quickly click "My Subreddits" and jump straight to a community I wish to view.

On "old" (Mlmym) Lemmy, my top bar is just Home and All. There is nothing to click to go straight to a community I wish to view.

On the regular ("new") Lemmy interface, I can click Communities > Subscribed to view my list of subscribed communities. But the order is strangely _random_, though, with no way to sort alphabetically. I have to "Find in page" to look for the community I wish to view.

Basically, when using the "new" Lemmy or "old" Lemmy style, there is **no** convenient way to view a list of my subscribed communities like I can with reddit.

Would it be possible to add either a drop-down list of (sorted) subscribed communities or some other page with (sorted) subscribed communities?

**Comment by Antik79** [comment]
Yes, this is definitely the most important feature for me currently. I'm sure that things like modlog and mod/admin tools are still far away but this would option would help a lot

We just added this to Lemmy World btw: Https://old.lemmy.world 😁
